### ISSUE TITLE
Using pre-trained model in DLC

### ISSUE THREAD

**Issue by HOLGUINsantiago** [mlfpm/deepof#58]
HI,

It's my first time usinf DeepOf and DLC with pre-trained models. I created a dlc project, I added data, label some of it, created a training dataset, but when i want to train the model from the initial weights shared in DeepOf documentation I get this error : 
Detected at node 'save/RestoreV2' defined at (most recent call last):
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\lib\runpy.py", line 196, in _run_module_as_main
      return _run_code(code, main_globals, None,
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\lib\runpy.py", line 86, in _run_code
      exec(code, run_globals)
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\Scripts\dlc.exe\__main__.py", line 7, in <module>
      sys.exit(main())
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\__main__.py", line 25, in main
      launch_dlc()
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\gui\launch_script.py", line 69, in launch_dlc
      sys.exit(app.exec_())
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\gui\tabs\train_network.py", line 244, in train_network    
      compat.train_network(config, shuffle, **kwargs)
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\compat.py", line 268, in train_network
      return train_network(
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\pose_estimation_tensorflow\training.py", line 262, in train_network
      train(
    File "C:\Users\santi\anaconda3\envs\DEEPLABCUT\lib\site-packages\deeplabcut\pose_estimation_tensorflow\core\train_multianimal.py", line 155, in train
      restorer = tf.compat.v1.train.Saver(variables_to_restore)
Node: 'save/RestoreV2'
2 root error(s) found.
  (0) NOT_FOUND: Key pose/pairwise_pred/block4/biases not found in checkpoint
         [[{{node save/RestoreV2}}]]
         [[save/RestoreV2/_79]]
  (1) NOT_FOUND: Key pose/pairwise_pred/block4/biases not found in checkpoint
         [[{{node save/RestoreV2}}]]
0 successful operations.
0 derived errors ignored.

**Comment by NoCreativeIdeaForGoodUserName** [comment]
Hi HOLGUINsantiago,

Thank you for trying out deepof. What your error shows me, is that the issue seems to be with the deeplapcut (DLC) package, as all of your error paths point to it. I assume you are struggling with training the tracking model in DLC to create mouse body part position data for every frame of your videos, am I correct?

The models in the deepof documentation are inteded to be applied on tables of tracked data **after** the DLC tracking is complete and all steps of the DLC workflow are finished. In general it is a separated two step workflow:

1. you first load your videos into DLC and create a model in DLC to track the bodyparts of your mice. All these steps are DLC internal and have nothing to do with DeepOF. The workflow is: 

**In**: video data and some labels -> **Out**: multiple data tables (one per video) that contain the position infomration for your mouse bodyparts in every video frame

2. Now after you have a batch of tables with tracked data, you are done with DLC. You can now create a project with deepof (as described in the tutorials) and analyse the tracked data. The workflow is:

**In**: multipple data tables (one per video) that contain the position information for your mouse bodyparts in every video frame (generated with e.g. DLC) **Out**: supervised or unsupervised behavior data. 

Respectively, I would refer you to the DLC issues thread: https://github.com/DeepLabCut/DeepLabCut/issues

If you have further questions, let me know. 

Best regards,
Patrick

**Comment by HOLGUINsantiago** [comment]
Hello Patrick,

Thank you very much for your response and for your time. I'm sorry for the misunderstanding. I did understand the work done by DeepOF; I had simply thought that the weights provided in the repository were meant to be used with DLC. I'll go ahead and train my own model then. Thank you again.

Best regards,
Santiago
